site/src/App.vue

```diff
@@ -146,7 +146,15 @@
       <div class="footer-sponsors">
         <router-link to="/sponsors"> View our sponsors</router-link>
       </div>
-      <div class="footer-updated">Last updated {{ lastUpdated }}</div>
+      <div class="footer-updated">
+        Last updated {{ lastUpdated }} (<a
+          :href="'https://github.com/quacs/quacs/commit/' + quacsHash"
+          >site</a
+        >,
+        <a :href="'https://github.com/quacs/quacs-data/commit/' + dataHash"
+          >data</a
+        >)
+      </div>
```

Comment:
Can you give these links `color: var(--raw-link)`?
I dont care if you use style tags or a class, but they are just hard to read on dark mode with the default color